### PR TITLE
UI/control groups kv

### PR DESCRIPTION
**Pull request by Monkeychip**
Addressing a ticket where the error message was not displaying correctly.  

error

### CODE REVIEW

File: ui/app/components/secret-edit.js [hunk 1]
```diff
@@ -197,26 +199,34 @@ export default Component.extend(FocusOnInsertMixin, WithNavToNearestAncestor, {
...
-              this.set(e, e.errors.join(' '));
-            });
-        } else {
-          this.saveComplete(successCallback, key);
+      })
+      .catch(error => {
+        if (error instanceof ControlGroupError) {
+          let errorMessage = this.controlGroup.logFromError(error);
+          this.set('error', errorMessage.content);
         }
```
Comment (chelshaw) [inline ui/app/components/secret-edit.js:228]: To keep the original functionality here, we should `throw error` if it's not an instance of control group error

File: ui/app/routes/vault/cluster/access/identity/show.js [hunk 2]
```diff
@@ -20,11 +20,6 @@ export default Route.extend({
     // if the record is in the store use that
     let model = this.store.peekRecord(modelType, params.item_id);
 
-    // if we don't have creationTime, we only have a partial model so reload
-    if (model && !model.get('creationTime')) {
```
Comment (chelshaw) [inline ui/app/routes/vault/cluster/access/identity/show.js:24]: Is this block part of this fix or a different bug? If part of the other bug we should describe it in the PR description

**Review by chelshaw — CHANGES_REQUESTED** [review]
Couple small changes, great work here! 💪

File: ui/app/routes/vault/cluster/access/identity/show.js (hunk 2 above)
Comment (Monkeychip) [reply ui/app/routes/vault/cluster/access/identity/show.js:24]: I'm actually going to remove it.  It's apart of a larger issue that has several other bugs attached to it. I've got the other bugs documented and will add this to the list.  Good call.

**Review by chelshaw — APPROVED** [review]

**Pull request merged**